### PR TITLE
Fix OznMon using oldest control file

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
This PR contains two small fixes for OznMon plots.  

1. Added ompstc8_n20 to the summary plot exclusion list.  Summary OznMon plots can only be generated for a few instruments and ompstc8 isn't one of them.
2. In OznMon plots added a check to avoid overwriting the control (*.ctl) files.  Without this check the control file is overwritten as the processing loop steps through the requested cycles in the plot.  This results in the oldest version of the control file being used.  That is both inefficient and means if there are recent changes to the control file, as there were to ompsnp_n20, they are not immediately picked up.  The new check fixes that.  The fix is in use on cactus since 9/23 and Haixia has confirmed the results are correct.

### CODE REVIEW

**Comment by EdwardSafford-NOAA** [comment]
@HaixiaLiu-NOAA @ADCollard if either of you has a moment could you review this PR?  It fixed a problem with the OznMon plots that Haixia spotted about a month ago.  Thanks.

**Review by RussTreadon-NOAA — APPROVED** [review]
Changes appear to be consistent with PR description.  I did not personally test the changes but the PR description states that change have been cycled and work as intended.

Approve pending approval from others who use OznMon output.

**Comment by HaixiaLiu-NOAA** [comment]
thanks @EdwardSafford-NOAA for fixing the oznmon issues. These changes have allowed the oznmon to generate correct plots.

**Review by HaixiaLiu-NOAA — APPROVED** [review]

**Pull request merged**
